src/runtime-rs/crates/hypervisor/src/qemu/cmdline_generator.rs

```diff
@@ -2657,6 +2657,11 @@ impl<'a> QemuCmdLine<'a> {
 
         result.append(&mut self.knobs.qemu_params().await?);
 
+        if self.config.boot_from_template {
+            result.push("-incoming".to_string());
+            result.push("defer".to_string());
+        }
+
```

Comment:
Please refer to the related implement of `memory` or `machine` or `kernel` or `knobs`, and refactor the template.